Doc/library/sqlite3.rst

```diff
@@ -771,6 +805,8 @@ Connection objects
 
    .. method:: create_aggregate(name, n_arg, aggregate_class)
 
+      :rtype: None
```

Comment:
Should this return be after the param list?

![image](https://github.com/hugovk/cpython/assets/1324225/d1c871bd-7a72-4017-976b-09eb441123cb)

https://hugovk-cpython--60.org.readthedocs.build/en/60/library/sqlite3.html#sqlite3.Connection.create_aggregate

Same for some others after this.

Reply:
> Should this return be after the param list?

Yup, good call.  Fixed in 55f63d3df4bd0663ebb7379c6600677c0775912b